docs/software/bittorrent.md

```diff
@@ -312,6 +424,54 @@ Also installs:
 
     Full list of `aria2.conf` settings: <https://aria2.github.io/manual/en/html/aria2c.html>
 
+=== "Service control"
+
+    Since Aria2 runs as systemd service, it can be controlled with the following commands:
+
+    ```sh
+    systemctl status aria2
+    ```
+
+    ```sh
+    systemctl start aria2
+    ```
+
+    ```sh
+    systemctl stop aria2
+    ```
+
+    ```sh
+    systemctl restart aria2
+    ```
+
+=== "View logs"
+
+    Logs can be viewed with the following commands:
+
+    ```sh
+    journalctl -u aria2
+    ```
+
+    ```sh
+    cat /var/log/aria2.log
+    ```
+
+    Issues with the web interface can be debugged via webserver logs:
+
+    ```sh
+    # With Lighttpd (default):
+    journalctl -u lighttpd
+    more /var/log/lighttpd/error.log
+
+    # With Apache:
+    journalctl -u apache2
+    more /var/log/apache2/error.log
+
+    # With Nginx:
+    journalctl -u nginx
+    more /var/log/nginx/error.log
+    ```
```

Comment:
Aria2 file logging will be disabled with next release (retroactively), and the default webserver is Apache:
```suggestion
    Issues with the web interface can be debugged via webserver logs:

    ```sh
    # With Apache (default):
    journalctl -u apache2
    more /var/log/apache2/error.log

    # With Nginx:
    journalctl -u nginx
    more /var/log/nginx/error.log

    # With Lighttpd:
    journalctl -u lighttpd
    more /var/log/lighttpd/error.log
    ```
```